src/osx/dataview_osx.cpp

```diff
@@ -233,76 +233,65 @@ void wxOSXDataViewModelNotifier::Resort()
 
 void wxOSXDataViewModelNotifier::AdjustRowHeight(wxDataViewItem const& item)
 {
-  if ((m_DataViewCtrlPtr->GetWindowStyle() & wxDV_VARIABLE_LINE_HEIGHT) != 0)
+  if (!m_DataViewCtrlPtr->IsFrozen() ||  ((m_DataViewCtrlPtr->GetWindowStyle() & wxDV_VARIABLE_LINE_HEIGHT) != 0))
```

Comment:
Is this condition correct, i.e. shouldn't it be an `&&`? IOW do we really need to do anything for a frozen control here?

Reply:
You're right, I think.

Reply:
Usage of `||` here results in the adjustments being delayed only if the view is frozen and not using variable row height, which corresponds with the commit message:
> If the view is Frozen and it's not using variable row heights, delay [...]

So with `wxDV_VARIABLE_LINE_HEIGHT` it behaves as before, I suppose either because it's still required or was only tested without `wxDV_VARIABLE_LINE_HEIGHT`.